src/pages/docs/dynamic-servers/concepts/docker-root-user.mdx

```diff
@@ -0,0 +1,83 @@
+# Docker root user
+
+Although Docker allows running containers as the root user, Rivet does not allow uploaded build images to use
```

Comment:
I think a lot of people will come in to this not understanding how linux users work/why they're important. This paragraph is good, but might be good to add a "learn more" link to something that describes users in dockerfiles in an easy to understand way.

Reply:
This whole document is _scary_ to someone who doesn't know how to use a cli. trying to make it less scary.